### ISSUE TITLE
Create "Quickstart" documentation

### ISSUE THREAD

**Issue by briantopping** [topolvm/topolvm#177]
## What

I filed #149 when it was late at night and I couldn't figure out how to install TopoLVM. Over weeks, I decided to not use it. Now I am back to try again. I see improvements of course, but I still cannot figure out how to do it easily. I am not stupid, so it must be very difficult. But writing documentation is hard, I understand that too.

The current documentation makes description on every detail. It says things like "if you need X, do Y". But some of these things X are difficult to know because the software has never been installed before. I'm sure these things will make sense on the second install, but they do not yet. This request is about growing the community, not a support request for my personal needs. 

I decided to just install it broken and debug it today. I want the functionality it provides and it's just going to take a lot of effort to do so. I'm not complaining about that, simply want to provide the feedback so others may have a more satisfying experience!

## How

I think there should be a "Quickstart" where everything is cut-and-paste. If defaults are not possible, maybe the scripts or the software should be upgraded to figure them out. It could use the dumbest least-efficient configuration possible. But if it works, then people can figure out how they want to improve next time. 

As an example, use the ["heredoc format"](https://stackoverflow.com/questions/2500436/how-does-cat-eof-work-in-bash) when necessary. It is okay to check examples into the source tree and tell people to check out the repository to apply them like `kubectl create -f examples/crds.yaml` if I can copy and paste that command. 

In other words, I don't care what I am installing, I trust your team with my cluster.

Please avoid the temptation to respond to this issue with "thanks, we accept PRs". I don't expect anybody to do anything for me, the software is great and it's enough. I am trying to help improve your customer adoption and if I could get the software running in the first place, I wouldn't need a QuickStart. 

Once it is clear how to install this, it is natural that someone will contribute a PR for #149. 

Thanks for the consideration! ♥️♥️

## Checklist

- [ ] Finish implementation of the issue
- [ ] Test all functions
- [ ] Have enough logs to trace activities
- [ ] Notify developers of necessary actions

**Comment by toshipp** [comment]
we have a helm chart now, and the example directory provides an easy demonstration.
